test_data/encoding.csv

```diff
@@ -4,7 +4,6 @@
 #
 # Format:
 #   latitude,longitude,length,expected code (empty if the input should cause an error)
-
```

Comment:
Python's tests can't handle blank lines, not sure about the other languages. I can update if we want to keep this, alternatively blank lines can be prefixed with `#`

Reply:
I think it's fine to remove it like you did.